kbase-extension/static/kbase/js/common/pythonInterop.js

```diff
@@ -244,16 +251,53 @@ define([], () => {
         return pythonCode;
     }
 
+    /**
+     * Builds the call to run_app_bulk
```

Comment:
On a different topic, presumably `run_app_batch` is the current (/old) way of doing things using the BatchApp, and `run_app_bulk` is the new way, using ee2's batch endpoint.

Reply:
Yeah, I don't want to just rewrite `run_app_batch` in tricky ways to maintain backward compatibility. I'd rather just deprecate it and make a new wrapper.